### PR TITLE
Fix: Users list is not alphabetically ordered on Admin pages

### LINKED ISSUE #22348: Users list on server admin page is not alphabetically ordered
**What happened**:
Users list is not alphabetically ordered by username.

**What you expected to happen**:
Users list should be alphabetically ordered by username

**How to reproduce it (as minimally and precisely as possible)**:
Goto:
Server Admin->Users

**Environment**:
- Grafana version: Latest
- Data source type & version: latest
- OS Grafana is installed on: MacOS
- User OS & Browser: Chrome
- Grafana plugins:
- Others:

### PR DESCRIPTION
**Pull request by vikkyomkar**
**What this PR does / why we need it**:
Users are listed in alphabetical order by their usernames on the server admin page.

**Which issue(s) this PR fixes**:

Fixes #22348

### CODE REVIEW

**Comment by vikkyomkar** [comment]
Hi @aknuds1 
Test **grafana-docker-ubuntu-pr** is failing with below error which doesn't seems to be related to the code change.

Can you please help me understand this ?

`E: The repository 'http://security.ubuntu.com/ubuntu cosmic-security Release' does not have a Release file.
E: The repository 'http://archive.ubuntu.com/ubuntu cosmic Release' does not have a Release file.
E: The repository 'http://archive.ubuntu.com/ubuntu cosmic-updates Release' does not have a Release file.
E: The repository 'http://archive.ubuntu.com/ubuntu cosmic-backports Release' does not have a Release file.`

**Comment by aknuds1** [comment]
Merge your branch with the latest Grafana master, that should fix the CircleCI issue.

**Comment by aknuds1** [comment]
I tested your branch and it doesn't affect the listing of users on the Configuration/Users page. My guess is the API endpoint you modified isn't used for that page.

**Comment by aknuds1** [comment]
I took care of merging your branch with master BTW; might as well do it since I needed to test your changes.

**Comment by vikkyomkar** [comment]
@aknuds1 This PR is for `admin/user` page [ http://localhost:3000/admin/users  ]
Can you pls review the changes ?

File: pkg/services/sqlstore/user.go [hunk 1]
```diff
@@ -454,7 +454,7 @@ func SearchUsers(query *models.SearchUsersQuery) error {
 	offset := query.Limit * (query.Page - 1)
 	sess.Limit(query.Limit, offset)
 	sess.Cols("u.id", "u.email", "u.name", "u.login", "u.is_admin", "u.is_disabled", "u.last_seen_at", "user_auth.auth_module")
-	sess.OrderBy("u.id")
+	sess.OrderBy("LOWER(u.email),u.email")
```
Comment (aknuds1) [inline pkg/services/sqlstore/user.go]: @torkelo Should we have the same ordering here as for org_users (`sess.Asc("user.email", "user.login")`)?

Comment (vikkyomkar) [reply pkg/services/sqlstore/user.go]: @aknuds1 Did you get a chance to review this change ?

Comment (aknuds1) [reply pkg/services/sqlstore/user.go]: @vikkyomkar I'm waiting for consensus on ordering in #22371, then I'll need to know if we should apply the same ordering here.

Comment (vikkyomkar) [reply pkg/services/sqlstore/user.go]: Hi @aknuds1 , now @torkelo has given his consent to this change pls refer  #22371

**Comment by aknuds1** [comment]
Closed in favour of #23328.

**Pull request closed without merging**